packages/docs/docs/getting-started/layouts.mdx

```diff
@@ -107,6 +107,65 @@ useful properties are listed below:
 <ApiSnippet url={'/api/2d/components/Layout#justifyContent'} />
 <hr />
 
+### Disabling `layout` on a child node
+
+Sometimes it can be useful to remove the node from the node tree. This way the
+node can be styled independently of its ancestor layout, as well as moved freely
+on the canvas.
+
+Removal of node from the node tree is done by setting a value of `layout`
+property to `false`. The node can be brought back to its original position
+inside the node tree by setting the value of `layout` property to `true`.
+
+After disabling `layout` on a node, the following things change:
+
+1. the node is moved to the place of origin, e.g. center of the scene,
+2. the node can now be position on canvas freely,
+3. the node no longer inherits properties from its parent, such as text styling,
+4. the node's former parent layout will no longer considers the node's width or
+   height when calculating its own bounding box. Disabling a layout on a node
+   can be roughly thought of as an equivalent to setting a `position` CSS
+   property of an HTML node to `absolute` value.
+
+:::info
+
+The node retains its rendering order (`z-index`, in terms of CSS), e.g. it will
+potentially overlap any other nodes that were rendered previously, as well as
+get potentially covered by nodes that appear later.
+
+:::
+
+Consider the following example:
+
+1. a top-level `<Rect>` node has a `layout` property set, meaning that all its
+   children nodes follow `<Rect>`'s styling and direction,
```

Comment:
I don't like the emphasis on "styling and direction" here.
The point of the `layout` property is to arrange the children into a layout by changing their sizes and postions.
Inheriting the text style is a secondary thing that's not yet described in other sections.
Additionally, this example doesn't even use text.